specification/schemas/shops/Shop.json

```diff
@@ -223,7 +223,7 @@
               "type": "string",
               "readOnly": true,
               "example": "https://my-return-portal-domain.com",
-              "description": "The URL used as the return portal for this shop. If not set, defaults to the return portal domain of the broker followed by the shop's name. E.g. `https://return.myparcel.com/my-shop`"
+              "description": "The URL used for the return portal for this shop. If not set, defaults to the return portal domain of the broker followed by the shop's name. E.g. `https://return.myparcel.com/my-shop`"
```

Comment:
Let's also put the example that's in the description as the example for this field!